src/Service/ActivityPub/ApHttpClient.php

```diff
@@ -337,13 +338,21 @@ private function logRequestException(?ResponseInterface $response, string $reque
             }
         }
 
-        $this->logger->error('{type} get fail: {address}, ex: {e}: {msg} - {content}', [
+        // Often 400, 404 errors just return the full HTML page, so we don't want to log the full content of them
+        // We truncate the content to 200 characters max.
+        $this->logger->error('{type} get fail: {address}, ex: {e}: {msg}. Truncated content: {content}', [
             'type' => $requestType,
             'address' => $requestUrl,
             'e' => \get_class($e),
             'msg' => $e->getMessage(),
-            'content' => $content ?? 'no content provided',
+            'content' => substr($content, 0, 200) ?? 'No content provided',
```

Comment:
```suggestion
            'content' => substr($content ?? 'No content provided', 0, 200),
```
Otherwise this will throw an exception when `$content` is `null`

Reply:
ow yes.. you're right. Good spot.